src/inflation/adapter/load_files.clj

```diff
@@ -0,0 +1,38 @@
+(ns inflation.adapter.load_files
+  (:require
+   [clojure.string :as str]
+   [datomic.api :as d]
+   [inflation.db.datomic.db :as db]
+   [inflation.model.price :as model]
+   [clojure.data.csv :as csv]
+   [clojure.java.io :as io]))
+
+(def conn (db/connection))
+
+(db/create-schema conn)
+
+(defn insert-entity [entity]
+  (prn entity)
+  (d/transact conn [entity]))
+
+(defn read-file [filename] 
+  (with-open [reader (io/reader filename)]
+    (doall
+     (csv/read-csv reader))))
+
+(defn create-price [item, country, type]
+  (let [value (bigdec (nth item 2))
+        month (long (Float/valueOf (nth item 1)))
+        year (long (Float/valueOf (nth item 0)))]
+    (model/new-price country type value year month)))
+
+(defn load-file [filename country type]
+  (doseq [item (read-file filename)]
+    (-> item 
+        (first)
+        (str/split #";")
+        (create-price country type)
+        (insert-entity))))
+
+(load-file "resources/br_salary.csv" "BR" "SALARY")
+(load-file "resources/br_basic_food_basket.csv" "BR" "BFB")
```

Comment:
Avoid keeping side effect things as top level forms. They will be run every time you evaluate the namespace. Prefer creating a function that is called when the application is started. If this is only for testing, wrap it in a `comment` form. Then you can run in the repl only when you need.